ad-first-campaign.py

```diff
@@ -0,0 +1,66 @@
+def main():
+
+    import sem
+    import math
+    import numpy as np
+    import os
+    import matplotlib
+    matplotlib.use('Agg')
+    import matplotlib.pyplot as plt
+    from collections import OrderedDict 
+
+    # Define campaign parameters
+    ############################
+
+    script = "evaluate_scheduler_qd_dense_scenario"
+    cmp_name = "basic-vs-cbaponly-v2"        # define a campaign name in order to create different corresponding to a specific configuration
+    ns_path = os.path.join(os.path.dirname(os.path.realpath(__file__)))
+    campaign_dir = "./campaigns/"+cmp_name
```

Comment:
be consistent with the style
```suggestion
    campaign_dir = "./campaigns/" + cmp_name
```